src/Symfony/Component/Config/Tests/Builder/Fixtures/AddToList/Symfony/Config/AddToList/Messenger/ReceivingConfig.php

```diff
@@ -8,12 +8,13 @@
 
 
 /**
- * This class is automatically generated to help creating config.
+ * This class is automatically generated to help in creating a config.
  */
 class ReceivingConfig 
 {
     private $priority;
     private $color;
+    private $_usedProperties = [];
```

Comment:
we don't use underscores in property names

Reply:
@nicolas-grekas This is to avoid collision with configuration proprieties (`->scalarNode('used_properties')`), same as with `_extraKeys`.

Sorry, I accidentally resolved this.

Reply:
Got it, thanks for the explanation.